pkg/controllers/managementuser/rbac/reconcile_roletemplate_test.go

```diff
@@ -198,58 +214,106 @@ func TestEnsureGlobalResourcesRolesForPRTB(t *testing.T) {
 				},
 			},
 		},
+		{
+			projectName: "testproject",
+			description: "error return when RoleTemplate client returns error",
+			manager: setupManager(
+				map[string]*v3.RoleTemplate{"create-ns": createNSRoleTemplate},
+				make(map[string]*v1.ClusterRole),
+				make(map[string]*v1.Role),
+				make(map[string]*v3.Project),
+				clientErrs{},
+				clientErrs{getError: errNotFound},
+				clientErrs{},
+			),
+			isErrExpected: true,
+			roleTemplates: map[string]*v3.RoleTemplate{
+				"testrt": {
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "testrt",
+					},
+					Rules: []v1.PolicyRule{
+						{
+							Verbs:     []string{"create"},
+							APIGroups: []string{""},
+							Resources: []string{"namespaces"},
+						},
+					},
+				},
+			},
+		},
+		// {
```

Comment:
If this test is no longer needed, it should be removed before merging.

Reply:
Yes, removed thanks! I had a chat with @pmatseykanets and the test case was unclear, nor correct.
Having a look when it was added (4479597c19bb4f7a74c304aa621932d213a05532) it was already failing, commenting the last test. Also the case of a "get role" failing seems to be not possible to have.